gossip/minimal/configpb/gen.go

```diff
@@ -15,4 +15,4 @@
 // Package configpb holds a config protobuf for minimal gossip binaries.
 package configpb
 
-//go:generate protoc -I=. -I=$GOPATH/src --go_out=:. config.proto
+//go:generate sh -c "protoc -I=. -I$(go list -f '{{ .Dir }}' github.com/google/trillian) --go_out=:$GOPATH/src config.proto"
```

Comment:
As discussed in person, `--go_out` still using `$GOPATH` means that certificate-transparency-go will have to be checked out under $GOPATH (i.e. it can't be outside of the Go source tree like modules usually can). We should document this restriction, since we're not currently aware of a way to avoid it.